### PR TITLE
Minor documentation fixes

### PR DESCRIPTION
**Pull request by roman-khimov**
Probably needs to be updated after #2827.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,27 +48,28 @@ NeoGo, `:latest` points to the latest release) or build yourself.
 
 ### Building
 
-To build NeoGo you need Go 1.17+ and `make`:
+Building NeoGo requires Go 1.17+ and `make`:
 
 ```
-make build
+make
```
Comment (AnnaShaleva) [inline README.md:54]: Wow, I wasn't aware of this behaviour.

**Review by AnnaShaleva — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -126,37 +131,39 @@ Refer to [consensus node documentation](docs/consensus.md).
 
 ## Smart contract development
 
-Please refer to [neo-go smart contract development
+Please refer to [NeoGo smart contract development
```
Comment (AnnaShaleva) [inline README.md:134]: If we're fixing this, then maybe we'd like to fix the `UserAgentPrefix = "NEO-GO:"` as far?

File: README.md [hunk 3]
```diff
@@ -126,37 +131,39 @@ Refer to [consensus node documentation](docs/consensus.md).
...
 workshop](https://github.com/nspcc-dev/neo-go-sc-wrkshp) that shows some
-simple contracts that can be compiled/deployed/run using neo-go compiler, SDK
+simple contracts that can be compiled/deployed/run using NeoGo compiler, SDK
```
Comment (AnnaShaleva) [inline README.md:136]: Not related to this line, but we also have `Usage:     "start a NEO node",` description for the `neo-go node` CLI command. Is it important?

Comment (AnnaShaleva) [reply README.md:136]: And `Usage: "create, open and manage a NEO wallet"` for `neo-go wallet` set of commands.

Comment (AnnaShaleva) [reply README.md:136]: And `Usage:     "convert addresses from existing NEO2 NEP6-wallet to NEO3 format",`.

Comment (AnnaShaleva) [reply README.md:136]: And `Usage:     "check and dump an existing NEO wallet",`.

Comment (AnnaShaleva) [reply README.md:136]: And `The neo-go compiler compiles Go programs to a bytecode that the NEO virtual machine can understand.` (this one is from documentation, so may be fixed in the current PR).

Comment (AnnaShaleva) [reply README.md:136]: And `Compiler translates interop function calls into NEO VM syscalls or (for custom functions) into NEO VM instructions.`.

Comment (AnnaShaleva) [reply README.md:136]: And `Magic number which uniquely identifies NEO network.`.

Comment (AnnaShaleva) [reply README.md:136]: And a couple of other NEO/Neo misuses.

**Review by AnnaShaleva — COMMENTED** [review]
I'm not sure whether we need to fix all of this things. If not, then LGTM.

File: README.md (hunk 2 above)
Comment (roman-khimov) [reply README.md:134]: I'm scared to even try changing, it was like that since the beginning.

File: README.md (hunk 3 above)
Comment (roman-khimov) [reply README.md:136]: Patches are welcome, this one is about README mostly and the bug we have.

Comment (AnnaShaleva) [reply README.md:136]: OK, then I'll create a separate PR with these changes.

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**